server/pension/models.py

```diff
@@ -21,13 +21,13 @@ class InstrumentType(object):
     TYPE2 = 2
 
     choices = (
-        (TYPE1, _('')),
-        (TYPE2, _('')),
+        (TYPE1, _('ניירות ערך סחירים')),
```

Comment:
@nirgn975 
Can you please provide the english expression for this one over here ?

@Lemore 
We shouldn't write any translated string on the code, we'll translate this one later on.